main/src/library/Graph.flix

```diff
@@ -118,4 +117,86 @@ namespace Graph {
         let selfReach = query edges, reachability select () from Reachable(x, y) where x == y;
         0 < (selfReach |> List.length)
 
+    ///
+    /// Wrapper type to invert lattice on `Int32`.
+    ///
+    enum InvertLattice {
```

Comment:
I would like to make the following changes:

- Rename `InvertLattice` to `Down` and move it to `Down.flix`.
- Refactor `Down` to work for any `a` that also has instances of `Eq`, `Order`, etc. (i.e. reversing the order).

Reply:
The latter means, e.g. `instance Order[Down[a]] with Order[a] { ... `.